src/components/marker-chart/index.js

```diff
@@ -115,8 +120,13 @@ function viewportNeedsUpdate(
 
 const options: ExplicitConnectOptions<{||}, StateProps, DispatchProps> = {
   mapStateToProps: state => {
-    const markers = selectedThreadSelectors.getTracingMarkersForView(state);
-    const markerTimingRows = selectedThreadSelectors.getMarkerTiming(state);
+    const isNetworkChart = getSelectedTab(state) === 'network-chart';
+    const markers = isNetworkChart
+      ? selectedThreadSelectors.getNetworkChartTracingMarkers(state)
+      : selectedThreadSelectors.getMarkerChartTracingMarkers(state);
+    const markerTimingRows = isNetworkChart
+      ? selectedThreadSelectors.getNetworkChartTiming(state)
+      : selectedThreadSelectors.getMarkerChartTiming(state);
```

Comment:
now that these values are filtered from the range and the selection, maybe we need to remove the filtering done inside `MarkerChartCanvas` too ?
https://github.com/devtools-html/perf.html/blob/4a1de1c56c3c71aabb6d53df3c8cf00258a71512/src/components/marker-chart/Canvas.js#L184-L188

Reply:
I believe that check works for the preview range, not just the committed range, so it is still needed.